soroban-sdk/src/accounts.rs

```diff
@@ -421,7 +435,11 @@ impl testutils::Accounts for Accounts {
                     last_modified_ledger_seq: 0,
                     ext: xdr::LedgerEntryExt::V0,
                 };
-                storage.put(&k, &v)
+                storage.put(
+                    &k,
+                    &v,
+                    soroban_env_host::budget::AsBudget::as_budget(env.host()),
```

Comment:
What does as budget do in these calls now?

Reply:
Logically same as before. It's just now a `Budget` is explicitly passed in every storage call, whereas previously the `Storage` owns a `MeteredOrdMap` which owns a `Budget` reference (same one as the host owns). Internally it charges the budget before preforming the corresponding map operation.